examples/image_classification/pytorch_classification.py

```diff
@@ -0,0 +1,60 @@
+import time
+from aperturedb import Connector
+import AlexNetClassifier as alexnet
+
+db = Connector.Connector(user="admin", password="admin")
```

Comment:
can we split this file into functions, have a main, and use dbinfo.py to create the connector?